src/tests/App.test.js

```diff
@@ -0,0 +1,27 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import data from './data';
+import App from '../App';
+
+const reducer = (
+  state = {
+    home: { home: data },
+  },
+) => state;
+
+const store = configureStore({ reducer });
+describe('App test', () => {
+  it('will test App rendering', () => {
+    render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <App />
+        </Provider>
+      </MemoryRouter>,
+    );
```

Comment:
- [ ] Dear @mariegrace31, Rendering the page by itself is not enough to test the app functionality. Please make use of the Jest to test several components of the app and some pure functions in the Redux code.
- [ ] Please also add some integration tests using React Testing Library, this could be used to simulate user interaction.